templates/web/base/report/display.html

```diff
@@ -20,6 +20,8 @@
     <p class='form-success'>[% loc('You have successfully signed in; please check and confirm your details are accurate:') %]</p>
 [% END %]
 
+[% TRY %][% INCLUDE 'report/_extras.html' %][% CATCH file %][% END %]
```

Comment:
Could this be called `_extras_top.html` or similar to signify it's included first thing?